src/Analyzers/CSharp/Tests/InvokeDelegateWithConditionalAccess/InvokeDelegateWithConditionalAccessTests_FixAllTests.cs

```diff
@@ -46,7 +46,6 @@ class C
                     void Goo()
                     {
                         a?.Invoke();
-
                         a?.Invoke();
```

Comment:
these changed because the code that was generated was `(a)?.Invoke();` with elastic trivia on the parens.  But when the parens were removed in the simplification pass, we lost that trivia.  So we weren't formatting properly.  the new formatting follows the expected formatting rules now.